pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-8239_created_created_by_cohort_tags.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
because this is a general table change and not just CMI related I think we should move it out of the CMI folder

Reply:
@SimoneMaiwald since cohort tag table liquibase is already run on production if we change file location we will struggle because `DATABASECHANGELOG` table already tracked that it was run from `CMI` directory, let's keep it like that for now.